spec/controllers/admin/entries_controller_spec.rb

```diff
@@ -155,8 +161,10 @@ def self.name
       expect(Pageflow::Entry.last.custom_field).to eq('some value')
     end
 
-    it 'does not allows account manager to define custom field not registered as form input' do
-      user = create(:user, :account_manager)
+    it 'does not allows account publisher to define custom field not registered as form input' do
+      user = create(:user)
+      account = create(:account)
+      create(:membership, user: user, entity: account, role: 'publisher')
```

Comment:
Using the `with_publisher_member` factory attributes would make this more concise.